lib/godwoken_explorer/graphql/resolovers/udt.ex

```diff
@@ -53,6 +54,25 @@ defmodule GodwokenExplorer.Graphql.Resolvers.UDT do
     {:ok, return}
   end
 
+  def token_exchange_rate(%UDT{} = udt, _args, _resolution) do
+    symbol = udt.symbol
+
+    if symbol do
```

Comment:
Is `symbol` enough to locate the official token? Say I have minted a token with the symbol `USDT`, will the price be returned along?

Reply:
yes, if you minted a token witch symbol also name 'USDT', it will return the price too.